relay-event-normalization/src/normalize/span/description/sql/parser.rs

```diff
@@ -492,7 +497,6 @@ impl VisitorMut for NormalizeVisitor {
                     }
                 }
             }
-
             _ => {}
```

Comment:
Can we get rid of this catch all, if in the future the library introduces new values to the enum it won't compile and we know we have to check something?

Reply:
We absolutely should. I already started doing it and then gave up because there are so many match arms. But I guess I'll just power through.